specific_use_cases/DFU_work_in_progress/utils_notebook_1.py

```diff
@@ -67,4 +67,13 @@ def hdd_cdd_lineplot(annual_data, trendline, title="title"):
         color="black", 
         line_dash='dashed', 
         label="trendline"
-    ) 
\ No newline at end of file
+    ) 
+
+def hdh_cdh_lineplot(data, date_start, date_end):
+    data_to_plot = data.loc[:, date_start:date_end]
```

Comment:
My preference would be for the plotting function to just perform the plotting, and have the time slice occur directly in the notebook. I also really like xarray's selection functionality that allows you to just use "June 2011" to get the same data you have here-- it's super easy to read/use and might empower users to play with it more? Just my two cents!

```
cdd_one_month = cdh_aggregated.sel(time="June 2011")
hdh_cdh_lineplot(cdd_one_month)
```

Reply:
Plus, I think the plotting function should work regardless of whether or not you want to plot a particular time slice